### PR TITLE
llvm=1

### PR DESCRIPTION
**Pull request by nickdesaulniers**

### CODE REVIEW

**Comment by nickdesaulniers** [comment]
LGTM: https://travis-ci.com/github/ClangBuiltLinux/continuous-integration/jobs/375549925#L521

File: driver.sh [hunk 1]
```diff
@@ -22,6 +22,13 @@ setup_variables() {
         shift
     done
 
+    set +u
+    if [[ -z "$ARCH" ]]; then
```
Comment (nathanchance) [inline driver.sh:26]: Could do `if [[ -z "${ARCH:-}" ]]; then` instead of the `set +u` dance.

File: driver.sh [hunk 2]
```diff
@@ -332,26 +339,38 @@ check_objcopy_strip_version() {
...
-        LD="${LD}" \
-        LLVM_IAS="${LLVM_IAS}" \
-        NM="${NM}" \
-        OBJCOPY="${OBJCOPY}" \
-        OBJDUMP="${OBJDUMP}" \
-        OBJSIZE="${OBJSIZE}" \
-        READELF="${READELF}" \
-        STRIP="${STRIP}" \
-        "${@}"
+    if [[ ${LLVM} -eq "1" ]]; then
```
Comment (nathanchance) [inline driver.sh:342]: Will probably need something like `${LLVM:-0}` due to `set -u`.

File: .travis.yml [hunk 3]
```diff
@@ -24,8 +24,8 @@ jobs:
       env: ARCH=riscv LLVM_IAS=1
     - name: "ARCH=s390 BOOT=false"
       env: ARCH=s390 BOOT=false
-    - name: "ARCH=x86 LD=ld.lld"
-      env: ARCH=x86 LD=ld.lld
+    - name: "ARCH=x86 LLVM=1"
```
Comment (nathanchance) [inline .travis.yml:27]: Let's get #302 merged then switch over as much as possible to `LLVM=1` + GNU variants as we need them.

File: driver.sh (hunk 1 above)
Comment (nickdesaulniers) [reply driver.sh:26]: `./driver.sh: line 25: ARCH: unbound variable`
am I holding it wrong? 🤡

Comment (nickdesaulniers) [reply driver.sh:26]: maybe default to zero and compare against a value of zero, rather than use `-z`?

File: driver.sh (hunk 2 above)
Comment (nickdesaulniers) [reply driver.sh:342]: yep, done

File: .travis.yml (hunk 3 above)
Comment (nickdesaulniers) [reply .travis.yml:27]: will do

File: driver.sh (hunk 1 above)
Comment (nathanchance) [reply driver.sh:26]: Yes, that is probably a better idea.

Comment (nickdesaulniers) [reply driver.sh:26]: done

**Pull request merged**